docs/products/flink/howto/ext-pg-flink-integration.md

```diff
@@ -0,0 +1,134 @@
+execute the following command:
+
+```bash
+avn service create -t flink -p <project-name> --cloud <cloud-name> <flink-service-name>
+```
+
+Alternatively, if can create a new Aiven for Apache Flink service,
+you can use the following command:
+
+```bash
+avn service create -t flink -p <project-name> --cloud <cloud-name> <flink-service-name>
+```
+
+Parameters:
+
+- `-t flink`: The type of service to create, which is Aiven for Apache
+  Flink.
+- `-p <project-name>`: The name of the Aiven project where the service
+  should be created.
+- `cloud <cloud-name>`: The name of the cloud provider on which the
+  service should be created.
+- `<flink-service-name>`: The name of the new Aiven for Apache Flink
+  service to be created. This name must be unique within the specified
+  project.
+
+### Gather external PostgreSQL details
+
+Ensure you have all the necessary details of your external PostgreSQL database,
+including any SSL certificates, if applicable.
+
+### Create an external PostgreSQL endpoint
+
+Create an integration endpoint for your external PostgreSQL database using the
+[avn service integration-endpoint-create](/docs/tools/cli/service/integration#avn_service_integration_endpoint_create)
+command with the required parameters.
+
+```bash
+avn service integration-endpoint-create --project <project-name> \
+    --endpoint-name <external-pg-endpoint-name> \
+    --endpoint-type external_postgresql \
+    --user-config-json '{"host":"<postgres_host>", \
+    "port":"<postgres_port>", "db_name":"<postgres_db>", \
+    "user":"<postgres_user>", "password":"<postgres_password>", \
+    "ssl":"<ssl_mode>", "sslrootcert":"<path_to_ssl_cert>"}'
+```
+
+Parameters:
+
+- `--project <project-name>`: The name of your Aiven project.
+- `--endpoint-name <external-pg-endpoint-name>`: The name for the new endpoint.
+- `--endpoint-type external_postgresql`: The type of endpoint to create.
+- `--user-config-json`: Configuration details of your PostgreSQL database in JSON format.
+
+### Verify integration with service
+
+After creating the integration between Aiven for Apache Flink and your external
+PostgreSQL database, verify that the
+integration has been created successfully, and create applications that
+use the integration.
+
+```bash
+  avn service integration-list --project <project-name> <flink-service-name>
+```
+
+Parameters:
+- `--project <project-name>`: The name of your Aiven project.
+- `<flink-service-name>`: The name of your Aiven for Apache Flink service.
+
+To create Aiven for Apache Flink applications, you will need the
+integration ID of the Aiven for Apache Flink service. Obtain the
+`integration_id` from the integration list.
+
+### Create Aiven for Apache Flink applications
+
+With the integration ID obtained, you're now ready to develop Aiven for Apache Flink
+applications that utilize the data from your external PostgreSQL database
+as either a source or a sink.
```

Comment:
We've already mentioned this at the top of the page and previous section, let's limit repetition?

```suggestion
```